sonar-core/src/test/java/org/sonar/core/issue/workflow/IssueWorkflowTest.java

```diff
@@ -139,7 +139,7 @@ public void fail_if_unknown_status_when_listing_transitions() {
   }
 
   @Test
-  public void do_automatic_transition() {
+  public void automatically_close_resolved_issue() {
```

Comment:
the test was badly named but it does not relate to the bug being fixed. Another test is missing.